src/server/req_jobobit.c

```diff
@@ -1010,8 +1009,7 @@ on_job_rerun(struct work_task *ptask)
 				/* need to have MOM delete her copy of the job */
 				preq = alloc_br(PBS_BATCH_DeleteJob);
 				if (preq) {
-					(void)strcpy(preq->rq_ind.rq_delete.rq_objname,
-						pjob->ji_qs.ji_jobid);
+					preq->rq_ind.rq_delete.rq_objname = strdup(pjob->ji_qs.ji_jobid);
```

Comment:
U need to check for strdup() (all changed strdup)

Reply:
okay, I agree